release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -539,6 +541,20 @@ For more information, see xref:../networking/metallb/about-metallb.adoc#about-me
 [id="ocp-4-10-builds"]
 === Builds
 
+* With this update, you can run entitled builds with `SharedSecret` objects as a Technology Preview feature. This feature relies on the newly-introduced a new Shared Resource driver and the Insights Operator to import {op-system-base} Simple Content Access (SCA) certificates. By using this feature, you can install entitled RPM packages during builds without the extra effort of copying your {op-system-base} subscription credentials and certificates into the builds' namespaces. (link:https://issues.redhat.com/browse/BUILD-274[BUILD-274])
```

Comment:
```suggestion
* With this update, you can run entitled builds with `SharedSecret` objects as a Technology Preview feature. This feature relies on the newly introduced Shared Resource driver and the Insights Operator to import {op-system-base} Simple Content Access (SCA) certificates. By using this feature, you can install entitled RPM packages during builds without the extra effort of copying your {op-system-base} subscription credentials and certificates into the builds' namespaces. (link:https://issues.redhat.com/browse/BUILD-274[BUILD-274])
```
- Per ISG section on compound words, "Do not use a hyphen after an adverb or adjective that ends in -ly because such terms are never ambiguous."
- Deleted `a new` because it seems to be a typo

Reply:
Updated. Thanks!